src/styles/blocks/body.scss

```
@@ -0,0 +1,105 @@
+.card {
+  width: $card-width;
+  box-sizing: border-box;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+
+  &__image {
+    display: flex;
+    justify-content: center;
+    margin-bottom: 40px;
+  }
+
+  &__image-img {
+    width: 160px;
+    height: 134px;
+  }
+
+  &__title {
+    margin-bottom: 4px;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__code {
+    margin-bottom: 16px;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $grey-text-color;
+  }
+
+  &__review {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__review-info {
+    align-items: center;
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+    height: 18px;
+  }
+
+  &__price-value {
+    font-weight: 400;
+    font-size: 12px;
+    line-height: 18px;
+    color: $grey-text-color;
+  }
+
+  &__price-amount {
+    font-weight: 700;
+    font-size: 16px;
+    line-height: 18px;
+    color: $main-text-color;
+  }
+
+  &__button {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    width: 166px;
```

100% of parents width
```suggestion
    width: 100%;
```